### PR TITLE
Translated Swipe Actions to German

### PR DESCRIPTION
**Pull request by dertuxmalwieder**
Busy weekend, huh? ;-)

### CODE REVIEW

File: IceCubesApp/Resources/Localization/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -158,21 +158,21 @@
...
+"settings.swipeactions.status.action.bookmark" = "Lesezeichen setzen";
+"settings.swipeactions.status.action.boost" = "Boosten";
+"settings.swipeactions.status.action.favorite" = "Favorisieren";
+"settings.swipeactions.status.action.none" = "keine";
+"settings.swipeactions.status.action.quote" = "Zitieren";
+"settings.swipeactions.status.action.reply" = "Antworten";
+"settings.swipeactions.status.leading" = "Nach rechts wischen";
+"settings.swipeactions.status.leading.left" = "links";
+"settings.swipeactions.status.leading.right" = "links";
+"settings.swipeactions.status.trailing" = " Nach links wischen";
```
Comment (te6-in) [inline IceCubesApp/Resources/Localization/de.lproj/Localizable.strings:172]: I think the leading space is unnecessary. (The English one needs to be fixed too.)

Comment (dertuxmalwieder) [reply IceCubesApp/Resources/Localization/de.lproj/Localizable.strings:172]: Not sure about that ... I won't remove spaces without being sure that they're not intended...?

Comment (te6-in) [reply IceCubesApp/Resources/Localization/de.lproj/Localizable.strings:172]: Yeah, I think they are not intended and should be removed.

With a space:
SCR-20230212-it5

Without:
SCR-20230212-iuc

Maybe it'll be okay to remove it when the original English string gets fixed.

Comment (dertuxmalwieder) [reply IceCubesApp/Resources/Localization/de.lproj/Localizable.strings:172]: Will do … ;-)

**Pull request merged**